src/Sinch/Voice/Calls/Call.cs

```diff
@@ -60,19 +61,19 @@ public sealed class Call
         /// <summary>
         ///     An object that can be used to pass custom information related to the call.
         /// </summary>
-        public object Custom { get; set; }
+        public JsonObject? Custom { get; set; }
```

Comment:
Why this change? The documentation mentions a `string`: https://developers.sinch.com/docs/voice/api-reference/voice/tag/Calls/#tag/Calls/operation/Calling_GetCallResult!c=200&path=custom&t=response

Reply:
In this case, maybe you can leave it as it was and fix the type only once in a separate PR?